Sources/XCHammer/XcodeTarget.swift

```diff
@@ -1255,7 +1255,7 @@ public class XcodeTarget: Hashable, Equatable {
     }()
 
     func extractLibraryDeps(map targetMap: XcodeTargetMap) -> [String] {
-        return transitiveTargets(map: targetMap, predicate: stopAfterNeedsRecursive)
```

Comment:
Does this lead to generator speed regressions? Since we call `transitiveTargets(map: targetMap, predicate: keepGoing)` in multiple places now we could look at partial computation or a dynamic programming solution?

Reply:
Yep as I mentioned in the description this def has bad time complexity compared to before, this improvement is being tracked separately and once it's ready I can this grep for this predicate and improve it in all places

I like the dynamic programming suggestion, will experiment with that